tests/test.js

```diff
@@ -1155,6 +1155,30 @@ module("mixpanel.track_links");
         svg.remove();
     });
 
+    asyncTest("accepts a DOM element as the query", 1, function() {
+        var link = $('<a>', {href: '#test'})[0];
+
+        mixpanel.track_links(link, "testing url property", {}, function() {
+            start();
+            ok(1===1, "track_links callback was fired");
+            return false;
+        });
+
+        simulateMouseClick(link);
+    });
+
+    asyncTest("accepts an iterable list of DOM elements (such as a jquery list) as the query", 1, function() {
+        var $link = $('<a>', {href: '#test'});
```

Comment:
Is there a way to make this test use a jQuery obj matching multiple elements, and simulate a click on one that's not the first in the list?

Also a test using results from `document.querySelectorAll()` might be a good idea, to ensure it works for those using DOM APIs directly instead of jQuery.